app/src/main/java/org/mozilla/focus/browser/cursor/CursorView.kt

```diff
@@ -36,6 +39,18 @@ class CursorView(context: Context, attrs: AttributeSet) : ImageView(context, att
         resetCountdown()
     }
 
+    fun updateCursorPressedState(event: KeyEvent) {
+        val remoteKey = RemoteKey.fromKeyEvent(event)
+        // For keyboard and emulator use.
```

Comment:
nit: Comments should generally always have whitespace above them. In this case, I would:
```kotlin
// For...
val remoteKey ==
if (remoteKey...
```

I personally rather group unrelated code under a comment (unless it's really confusing) than "double-space" the code, but I can see the argument for the other direction.